bindings/node.js/kzg.cxx

```diff
@@ -173,31 +224,45 @@ Napi::Value ComputeKzgProof(const Napi::CallbackInfo& info) {
   KZGProof proof;
   C_KZG_RET ret = compute_kzg_proof(
     &proof,
-    (Blob *)blob,
-    (Bytes32 *)z_bytes,
+    blob,
+    z_bytes,
     kzg_settings
   );
 
   if (ret != C_KZG_OK) {
      Napi::Error::New(env, "Failed to compute proof")
       .ThrowAsJavaScriptException();
-    return env.Undefined();
+    return env.Null();
   };
```

Comment:
Are these semicolons after the brackets actually necessary in C++?

Reply:
Nope.  Was me and my predecessor semicoloning all the things!!!

I made another full pass and removed a couple other extras I found hanging out